src/Symfony/Component/Yaml/Inline.php

```diff
@@ -107,6 +109,10 @@ public static function parse($value, $flags = 0, $references = array())
                 $result = self::parseScalar($value, $flags, null, $i, true, $references);
```

Comment:
@xabbuh tags aren't supported on scalars yet anyway but updated to make sure we don't forget it in 4.0. thanks!